ui/eureka/eureka.js

```diff
@@ -82,7 +108,7 @@ let Options = {
             de: 'Sabo',
             fr: 'Pampa',
             ja: 'サボテン',
-            ko: '사보텐더',
+            ko: '사보',
```

Comment:
This looks like the changes here to the mob list will conflict with Katt's #903.  Can you verify that Katt's changes are identical so that this won't conflict?

Reply:
That looks identical with mine (for Korean 'trackerName's)